tests/components/tasmota/test_common.py

```diff
@@ -108,22 +108,6 @@
 }
 
 
-async def remove_device(hass, ws_client, device_id, config_entry_id=None):
```

Comment:
I felt it was better to remove this helper method completely, to align with #116240 and #116442

Another possibility is to adjust this function to have
```python
    if config_entry_id is None:
        config_entry_id = hass.config_entries.async_entries(DOMAIN)[0].entry_id
    response = await ws_client.remove_device(device_id, config_entry_id)
    assert response["success"]
```